cloud-sql/sql-server/sqlalchemy/Dockerfile

```diff
@@ -45,7 +45,8 @@ COPY . ./
 COPY ./certs /app/certs
 
 # Use server certificate for encrypted connection.
-COPY ./certs/server-ca.pem /usr/local/share/ca-certificates/server-ca.crt
+COPY Dockerfile ./certs/server-ca.pem* /usr/local/share/ca-certificates/
+RUN mv /usr/local/share/ca-certificates/server-ca.pem /usr/local/share/ca-certificates/server-ca.crt; exit 0
```

Comment:
Why is this rename here? Does using server-ca.pem break somewhere?

Reply:
This addresses the case where the ` server-ca.pem` file is not provided.
The first line conditionally copies the ` server-ca.pem` file and the second line renames the copied file if it exists.

Reply:
In that case, a comment explaining why this is happening be useful.

Reply:
Done.